icalBuddyDefines.h

```diff
@@ -171,7 +172,7 @@ typedef struct
     NSUInteger maxNumNoteCharacters;
 } AppOptions;
 
-#define NEW_DEFAULT_APP_OPTIONS {NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,\
+#define NEW_DEFAULT_APP_OPTIONS {NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,NO,\
                                  nil,nil,nil,nil,nil,nil,nil,nil,nil,nil,nil,nil,nil,0,0}
```

Comment:
Couldn't this just be {0}?

Reply:
The onlyalldayevents switch is a long standing wish of icalbuddy users! Thank you!